PSICover/PSICover.cs

```diff
@@ -150,6 +151,7 @@ class Analyzer {
    void GenerateOutputs () {
       ulong[] hits = File.ReadAllLines ($"{Dir}/hits.txt").Select (ulong.Parse).ToArray ();
       var files = mBlocks.Select (a => a.File).Distinct ().ToArray ();
+      Dictionary<string, (string csFilename, int covered, int total, double percentageCovered)> summary = new ();

```

Comment:
I think we don't need Dictionary here. Input is not high and all we are doing with summary is adding and sorting and for that list is sufficient.

Reply:
Done.